io.github.thetumultuousunicornofdarkness.cpu-x.yaml

```diff
@@ -0,0 +1,68 @@
+app-id: com.github.thetumultuousunicornofdarkness.cpu-x
+runtime: org.gnome.Platform
+runtime-version: "45"
+sdk: org.gnome.Sdk
+command: cpu-x
+
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # OpenGL/Vulkan
+  - --device=dri
+  # Needs to save files locally
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
really? The comment lead to this not doing what you think it is doing.

Reply:
Aaaah. I admit I took an existing YAML manifest from a different application (as skeleton) and I forgot to update this comment, sorry.
Improved in 5e2ec439e941f54accd6e9daa1a45b85a68962aa.